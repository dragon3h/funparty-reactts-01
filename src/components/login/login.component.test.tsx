import React from 'react';
import * as renderer from 'react-test-renderer';

import Root from '../../root';
import Login from './login.component';

it('renders Login component', () => {
  const tree = renderer
      .create(
          <Root>
            <Login />
          </Root>)
      .toJSON();
  
  expect(tree).toMatchSnapshot();
});
