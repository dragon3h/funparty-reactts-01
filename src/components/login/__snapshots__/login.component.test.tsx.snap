// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Login component 1`] = `
<div
  className="login"
>
  <h1>
    Login
  </h1>
  <form
    className="login-form"
    onSubmit={[Function]}
  >
    <div
      className="input-group"
    >
      <input
        className="form-input"
        name="email"
        onChange={[Function]}
        required={true}
        type="email"
        value=""
      />
      <label
        className=" form-input__label"
      >
        Email
      </label>
    </div>
    <div
      className="input-group"
    >
      <input
        className="form-input"
        name="password"
        onChange={[Function]}
        type="password"
        value=""
      />
      <label
        className=" form-input__label"
      >
        Password
      </label>
    </div>
    <button
      className="custom-button"
      type="submit"
    >
      Login
    </button>
  </form>
</div>
`;
